src/python/pants/option/global_options.py

```diff
@@ -265,6 +265,10 @@ def register_bootstrap_options(cls, register):
     # without resorting to heavier options parsing.
     register('--enable-pantsd', advanced=True, type=bool, default=False,
              help='Enables use of the pants daemon (and implicitly, the v2 engine). (Beta)')
+    register('--allow-pantsd-concurrent-runs', type=bool, default=False,
```

Comment:
I think I'd rather this be unrelated to pantsd, in particular it looks like if I add `--allow-pantsd-concurrent-runs` to my CLI, I will have the ability to run pantsd multiple times simultaneously, which is not the case.

I think I'd rather have the semantics be "I want to be able to run pants in parallel, please do what you must", which essentially ends up being turning pantsd off.

Reply:
I think the help for the option might make it clear what's being allowed? It could perhaps be renamed `--allow-concurrent-runs-under-pantsd`?

> I think I'd rather have the semantics be "I want to be able to run pants in parallel, please do what you must", which essentially ends up being turning pantsd off.

This is what happens now!

Reply:
how about `--pants-concurrent-runs` defaulted to False here? agree with Borja, rather not mention anything to do with `pantsd` here.